dashboard/src/actions/charts.ts

```diff
@@ -38,6 +46,14 @@ export const selectValues = createAction("SELECT_VALUES", resolve => {
   return (values: string) => resolve(values);
 });
 
+export const receiveChartUpdate = createAction("RECEIVE_CHART_UPDATES", resolve => {
+  return (chartUpdate: IChartUpdateCheck) => resolve(chartUpdate);
```

Comment:
the name of this type is confusing and it seems it's only used for this, I would just define an explicit type here of `{ name: string, update: IChartUpdate }`, or use 2 arguments.

Reply:
I am now using it in more places due to https://github.com/kubeapps/kubeapps/pull/930/files#r250379836